src/HermiteNormalForm.jl

```diff
@@ -1,38 +1,38 @@
 module HermiteNormalForm
 
 using LinearAlgebra
-export hnf!, hnf, ishnf
+export hnf!, hnf, ishnf, smith!, smith
```

Comment:
Do we have Smith form computation? :p

Reply:
No

Reply:
At one point I was playing around and added an incorrect `smith` function.